specification.markdown

```diff
@@ -968,7 +968,7 @@ A Global Element can be found in many Parent Elements, allowing the same number
 
 name: CRC-32
 
-path: `(1*\)\CRC-32`
+path: `\(1*\)\CRC-32`
```

Comment:
The idea is that the parent `\` is not part of the element name anymore, ie `\(1*\)CRC-32` or `\(*\)Void`. So if you read it like a pattern for CRC-32 you need at least one extra \ in the path. For Void it can have 0 (or plenty) extra \ in the path.